src/main/kotlin/com/cognifide/gradle/aem/base/vlt/VltCleaner.kt

```diff
@@ -71,57 +132,123 @@ class VltCleaner(val project: Project, val root: File) {
             logger.info("Cleaning file {}", file.path)
 
             val inputLines = FileUtils.readLines(file, CharEncoding.UTF_8)
-            val filteredLines = filterLines(file, inputLines, dotContentProperties)
+            val filteredLines = filterLines(file, inputLines)
 
             FileUtils.writeLines(file, CharEncoding.UTF_8, filteredLines, config.vaultLineSeparatorString)
         } catch (e: IOException) {
             throw VltException(String.format("Error opening %s", file.path), e)
         }
     }
 
-    private fun filterLines(file: File, lines: List<String>, props: List<VltContentProperty>): List<String> {
+    private fun filterLines(file: File, lines: List<String>): List<String> {
         val result = mutableListOf<String>()
-
         for (line in lines) {
             val cleanLine = StringUtils.trimToEmpty(line)
```

Comment:
type of lines is List<String> so that kotlin checks it about nullability, so that this util does not make sense here